Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Controllers/EmployeesController.cs

```diff
@@ -15,17 +15,25 @@
 namespace Benchmark.WebApi.Controllers
 {
     [Route("api/[controller]")]
-    [Authorize]
+    [AllowAnonymous]
```

Comment:
just make sure to remove this before it goes into develop
```suggestion
    [Authorize]
```